src/Illuminate/Database/Query/Grammars/PostgresGrammar.php

```diff
@@ -20,6 +21,43 @@ class PostgresGrammar extends Grammar
         'is distinct from', 'is not distinct from',
     ];
 
+    /**
+     * {@inheritdoc}
+     *
+     * @param Builder $query
+     * @param array   $where
+     *
+     * @return string
+     */
+    protected function whereBasic(Builder $query, $where)
+    {
+        // Special treatment for like clauses. In order to make
+        // any column searchable, we need to convert it to text.
+        if (Str::contains($where['operator'], 'like')) {
+            return $this->whereBasicLike($query, $where);
+        }
+
+        $value = $this->parameter($where['value']);
```

Comment:
You can use `return parent::whereBasic($query, $where);`.

Reply:
If this is the case there is no point in overwriting the method at all.

Reply:
How do you mean? I meant that the lines 40 and 42 do exactly what the parent method does.

Reply:
Sorry, you pointed the wrong lines, forget what I said you are right.

Reply:
My comment points to line 40, but GitHub also shows the previous lines.

Reply:
Changed to parent call.